IPython/lib/pretty.py

```diff
@@ -99,7 +99,7 @@ def _repr_pretty_(self, p, cycle):
 MAX_SEQ_LENGTH = 1000
 # The language spec says that dicts preserve order from 3.7, but CPython
 # does so from 3.6, so it seems likely that people will expect that.
-DICT_IS_ORDERED = sys.version_info >= (3, 6)
+DICT_IS_ORDERED = True
```

Comment:
Once this is merge, then the next step is to look at where `DICT_IS_ORDERED` is defined, and simplify that. But we can leave that for another day, or another new contributor.